crt_portal/cts_forms/locale/es/LC_MESSAGES/django.po

```diff
@@ -2812,6 +2814,20 @@ msgstr ""
 "acceder a esta página en otra pestaña o ventana en su navegador. Eso le "
 "creará otra cookie de seguridad y debería resolver el problema."
 
+#~ msgid "The .gov means it’s official."
+#~ msgstr "El .gov significa que es oficial."
+
+#~ msgid ""
+#~ "Federal government websites often end in .gov or .mil. Before sharing "
+#~ "sensitive information, make sure you’re on a federal government site."
+#~ msgstr ""
+#~ "Las páginas web federales suelen acabar en .gov o .mil. Antes de "
+#~ "compartir información confidencial, compruebe que se encuentra en un "
+#~ "sitio del gobierno federal."
+
+#~ msgid "The site is secure."
+#~ msgstr "Este sitio es seguro."
```

Comment:
Any idea why the compile just commented this out?  Should we keep it?

Reply:
It happened automatically during `makemessages`, and my guess is that it does this to allow fuzzy translations based on previously translated text even if the original stops existing in exact form. I don't think we have anything to lose by removing it.